src/Util/Search.php

```diff
@@ -765,4 +790,29 @@ private function doesUdiExistInQueryTerm(string $queryTerm, Query\BoolQuery $fie
         }
         return $queryTerm;
     }
+
+    /**
+     * Split the query terms into must match and must not match terms.
+     *
+     * @param string $queryTerm Query term that needs to be searched upon.
+     *
+     * @return array
+     */
+    private function splitQueryTerms(string $queryTerm): array
+    {
+        $splitUpQueryTerms = array();
+        if (preg_match_all('/-\b(\w*)\b/', $queryTerm, $matches)) {
+            $splitUpQueryTerms = array(
+                'mustNotMatch' => '',
+                'mustMatch' => ''
+            );
+            foreach ($matches[1] as $match) {
+                $splitUpQueryTerms['mustNotMatch'] = trim($splitUpQueryTerms['mustNotMatch'] . ' ' . $match);
+            }
+            $splitUpQueryTerms['mustMatch'] = str_replace('-', '', $queryTerm);
```

Comment:
I think this is maybe a tad too aggressive, removing embedded-between-words instead of just -excludes.  For example, a title contained the string 'July-October' which didn't work as expected when trying to exclude it. I think adding the word boundry as above would fix.

Reply:
Fixed it